src/action/base/setup_default_profile.rs

```diff
@@ -104,9 +115,18 @@ impl Action for SetupDefaultProfile {
             )));
         };
 
-        {
-            let reginfo_path =
-                Path::new(crate::action::base::move_unpacked_nix::DEST).join(".reginfo");
+        let found_nix_paths = glob::glob(&format!("{}/nix-*", unpacked_path.display()))
+            .map_err(|e| ActionError::Custom(Box::new(e)))?
+            .collect::<Result<Vec<_>, _>>()
+            .map_err(|e| ActionError::Custom(Box::new(e)))?;
+        assert_eq!(
```

Comment:
These panic-likes still have the autogenerated report URL, right? If not, might want to make an actual error for this.